### ISSUE TITLE
No files specified to analyze

### ISSUE THREAD

**Issue by me21** [netromdk/vermin#263]
I analyze my project using vermin and I found that `--no-make-paths-absolute` makes vermin unable to detect files in the most simple invocation.

I try: `vermin --no-make-paths-absolute .` to analyze the current directory, and I get `No files specified to analyze!` error. If I go up a level and execute `vermin --no-make-paths-absolute my_dir`, then it works. It also works if I remove `--no-make-paths-absolute` option.

**Environment (please complete the following information):**
 - Vermin version 1.6.0

**Comment by me21** [comment]
Thank you. Well, "." directory is not hidden, it's just the current directory, but okay.

**Comment by netromdk** [comment]
I agree that the distinction isn't fully covered correctly. But this will at least illuminate the issue when it occurs for now.
